packages/@sanity/portable-text-editor/src/utils/getPortableTextMemberSchemaTypes.ts

```diff
@@ -73,9 +73,9 @@ function resolveEnabledDecorators(spanType: ObjectSchemaType) {
 }
 
 function resolveEnabledListItems(blockType: ObjectSchemaType) {
-  const listField = blockType.fields?.find((btField) => btField.name === 'list')
+  const listField = blockType.fields?.find((btField) => btField.name === 'listItem')
```

Comment:
this is strictly speaking a breaking change, right? How likely is it that someone has `list` but not `listItem` and will get this error after upgrading?

Reply:
No, as this isn't really affecting the user schemas. They stay exactly the same. This is more the internal structure of the schema.